### PR TITLE
Link accessibility update

### PR DESCRIPTION
**Pull request by mbgower**
Added 3 images and new text content to accessibility tab

### CODE REVIEW

**Review by tw15egan — APPROVED** [review]
Just need to run `yarn format` to fix the failing tests, other changes look great 👍🏻

**Review by dakahn — APPROVED** [review]

File: src/pages/components/link/accessibility.mdx [hunk 1]
```diff
@@ -17,324 +17,93 @@ import {
...
+## Developer considerations
+
+Keep these considerations in mind if you are modifying Carbon or creating a
+custom component:
+
+- Associate generic links such as “read more” with other contextual text, using
+  either `aria-describedby` or `aria-labelledby` (to concatenate multiple text
+  strings). See the
+  [Equal Access Toolkit guidance](https://www.ibm.com/able/toolkit/develop/text-and-non-text/#aria-labelling)
+  for more details.
```
Comment (laurenmrice) [inline src/pages/components/link/accessibility.mdx:107]:
```suggestion
  for more details
```

**Review by laurenmrice — CHANGES_REQUESTED** [review]
Looks good! Just a couple content things to edit:

File: src/pages/components/link/accessibility.mdx [hunk 2]
```diff
@@ -17,324 +17,93 @@ import {
...
+![example of link keyboard interaction](images/link-accessibility-1.png)
 
-- [W3C WAI-ARIA Authoring Practices Link Design Pattern](https://www.w3.org/TR/wai-aria-practices/#link)
-  covers the usage of ARIA names, state and roles, as well as the expected
-  keyboard interactions.
-- [IBM Accessibility Checklist - Checkpoint 2.4.4 Link Purpose (In Context)](https://www.ibm.com/able/requirements/requirements/#2_4_4)
-  (WCAG Success Criteria
-  [2.4.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html))
+<Caption>
+  Links are reached by `Tab` key and activated by `Space` and `Enter` keys.
```
[the remark this answers is not in the document]
Comment (mbgower) [reply src/pages/components/link/accessibility.mdx]: done

**Review by laurenmrice — APPROVED** [review]

**Pull request merged**